.docker/Dockerfile

```diff
@@ -1,17 +1,10 @@
-ARG PHP_VERSION="7.3"
-
-FROM php:${PHP_VERSION}-cli
+ARG PHP_VERSION="8.1"
+ARG BUILD_TYPE="fpm"
 
+FROM arquivei/php:${PHP_VERSION}-${BUILD_TYPE}-debian as base
```

Comment:
Why have you changed the reference from official php image to an arquivei one? As far as I remember we've decided to start ending referencing it.

Reply:
You're right. I fixed this in the most recent commit.